### ISSUE TITLE
Phone number formatter doesn't work in Vaadin 23.3.x

### ISSUE THREAD

**Issue by syampillai** [johannesh2/textfieldformatter#65]
Seems to have problems with Vaadin 23.3.x

The following code throws javascript errors on the console looking for `phone-type-formatter.{country}.js`.

```

TextField textField;
add(textField = new TextField("Phone"));
new PhoneI18nFieldFormatter("US").extend(textField);
```

**Comment by johanneshayry** [comment]
Seems that maybe at a point the google libphonenumber was a transitive dependency, but not anymore. I need to release a new version with that dependency.

**Comment by johanneshayry** [comment]
No, I'm mistaken. The add-on is still included, but there is some other issue. I'll see if I can fix it.
